mobile/docs/root/api/starting_envoy.rst

```diff
@@ -369,33 +369,61 @@ Specify whether to use Happy Eyeballs when multiple IP stacks may be supported.
   // Swift
   builder.enableHappyEyeballs(true)
 
-~~~~~~~~~~~~~~~~~~~~~~~
-``enableGzip``
-~~~~~~~~~~~~~~~~~~~~~~~
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+``enableGzipDecompression``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 Specify whether to enable transparent response Gzip decompression. Defaults to true.
 
 **Example**::
 
   // Kotlin
-  builder.enableGzip(false)
+  builder.enableGzipDecompression(false)
 
   // Swift
-  builder.enableGzip(false)
+  builder.enableGzipDecompression(false)
```

Comment:
Optional, should we have a page on compile options and supported APIs somewhere? Maybe as a follow-up in the compile out PR?